osu.Game.Rulesets.Osu.Tests/Resources/special-skin/skin.ini

```diff
@@ -1,4 +1,7 @@
 [General]
 Version: latest
 HitCircleOverlayAboveNumber: 0
-HitCirclePrefix: display
\ No newline at end of file
+HitCirclePrefix: display
+
+[Colours]
+ReplayAimLine: 0,0,255
```

Comment:
We're not adding new items to `skin.ini` in general.

Reply:
How should I go about making the aim line color changeable? Or should it not be changeable in the first place?

Reply:
Non-changeable sounds fine for an initial impl.